ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+---
+- name: Choose a master on which to run
+  set_fact:
+    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
+  run_once: true
+
+- oc_obj:
```

Comment:
It would be nice to have a `name` here. Maybe something like "Get the host's pod network subnet".